EIPS/eip-X-ASIC-resistant-block-mining.md

```diff
@@ -0,0 +1,180 @@
+---
+eip: <to be assigned>
+title: Poisoning the Well
+author: David Stanfill <david@airsquirrels.com> 
+discussions-to: <email address>
+status: Draft
+type: Standards Track
+category Core
+created: 2018-04-03
+---
+
+
+## Simple Summary
+
+This EIP attempts to break ASIC miners specialized for the current ethash
+mining algorithm.
+
+
+## Abstract
+
+There are companies who currently have dedicated hardware based ethereum miners in
+production, and probabalistically actively mining.  This EIP aims to "Poison
+the well" by modifying the block mining algorithm in a manner that
+probabalistically *"breaks"* these miners if they are in-fact built on ASICs.
```

Comment:
Recommend rewording this to:
> modifying the proof of work algorithm in a manner that _may_ break existing ASICs.

Once again, EIPs are not the place for speculation, and until we know exactly how ASICs are functioning we cannot assert with any confidence whether or not a particular change will break anything.  For all we know, the ASICs being sold are just second hand GPUs mounted together in a box.

Reply:
not at 800$ buddy.. we know for certain they are not GPUs mounted together in a box!